scripts/setup-olm.sh

```diff
@@ -76,14 +77,22 @@ then
     $OPERATOR_SDK olm uninstall || true
 fi
 
+COUNT=$(kubectl get namespace openshift-operator-lifecycle-manager -o yaml 2>/dev/null | grep config.openshift.io/v1 | wc -l)
```

Comment:
We have duplicated this check.  Can we add a script, call it `is-openshift.sh` and just call the script to remove the duplication?

The `if` check then becomes:  `if $SCRIPT_DIR/is-openshift.sh`

Reply:
ok